genai-perf/tests/test_llm_profile_data_parser.py

```diff
@@ -200,21 +201,140 @@ def test_triton_llm_profile_data(
         * input sequence lengths
             - experiment 1: [3, 4]
             - experiment 2: [3, 4]
+        * request goodputs
+            - experiment 1: [1 / (11 - 1)] = [1 / 10]
+            - experiment 2: [0 / (18 - 3)] = [0]
         """
         tokenizer = get_tokenizer(DEFAULT_TOKENIZER)
+        test_goodput_constraints = {
+            "time_to_first_token": 2.5e-6,  # ms
+            "inter_token_latency": 2.5e-6,  # ms
+            "output_token_throughput_per_request": 0.5e9,  # s
+        }
         pd = LLMProfileDataParser(
             filename=Path("triton_profile_export.json"),
             tokenizer=tokenizer,
+            goodput_constraints=test_goodput_constraints,
         )
 
-        statistics = pd.get_statistics(infer_mode=infer_mode, load_level=load_level)
-        metrics = cast(LLMMetrics, statistics.metrics)
+        # experiment 1 metrics & statistics
+        stat_obj = pd.get_statistics(infer_mode="concurrency", load_level="10")
+        metrics = stat_obj.metrics
+        stat = stat_obj.stats_dict
 
-        expected_metrics = LLMMetrics(**expected_metrics)
-        expected_statistics = Statistics(expected_metrics)
+        assert isinstance(metrics, LLMMetrics)
 
-        check_llm_metrics(metrics, expected_metrics)
```

Comment:
Why do you need to add all of the asserts instead of using the check functions? It's often a bad sign if we're making a test a lot longer, especially if the changes do not seem relevant to the feature (i.e. goodput).

Reply:
I believe this is a recently merged update made by other people. I will solve the conflicts or reuse the function accordingly.

Reply:
Ah, I see. Thanks!